src/agents/pnp/CMakeLists.txt

```diff
@@ -80,7 +80,8 @@ else()
 
     target_include_directories(${target_name} PUBLIC
         ${SHARED_UTIL_INC_FOLDER}
-        ${MACRO_UTILS_INC_FOLDERP})
+        ${MACRO_UTILS_INC_FOLDERP}
+        ${PLATFORM_INC_DIR})

```

Comment:
> ${PLATFORM_INC_DIR}) [](http://example.com/codeflow?start=7&length=21)

Is this really required here for Percept? It's not needed on Ubuntu and Debian.

Reply:
its needed if we are using `azure-iot-sdk` as package, regardless of the distro